prime-dotnet-webapi/Views/Agreements/CommunityPharmacyOrganizationAgreement.cshtml

```diff
@@ -586,7 +586,12 @@
   </li>
 </ol>
 
-<p class="text-center">
+@if (@Model.WithSignature)
```

Comment:
Just want to confirm that they want the singature block before the Schedule A section?

Reply:
Yep